llvm/lib/Target/EVM/MCTargetDesc/EVMAsmBackend.cpp

```diff
@@ -138,7 +138,9 @@ bool EVMAsmBackend::evaluateTargetFixup(const MCAssembler &Asm,
   Value = Target.getConstant();
   if (const MCSymbolRefExpr *A = Target.getSymA()) {
     const MCSymbol &Sym = A->getSymbol();
-    assert(Sym.isDefined());
+    if (!Sym.isDefined())
```

Comment:
Doesn't seem to be relevant to the commit introducing the change. Perhaps, it's better to extract to a separate fixup.

Reply:
Removed.